### PR TITLE
Undraft the bylaws: content templates

### PR DESCRIPTION
**Pull request by ebullient**
[🗳️ Vote progress](https://github.com/commonhaus/foundation/pull/173#issuecomment-2183217739 "IC_kwDOKRPTI86CIUJL")

These templates provide suggestions for a website footer, GOVERNANCE.md file, and language surrounding use of DCO or CLA.

All of this is in the realm of recommendation, rather than requirement, but establishes the flavor of what we're looking for.

voting group: @commonhaus/cf-egc 

> [!TIP]
>  Use the +- button to make a suggestion (instead of just adding a comment) to keep things moving forward.

Do one of the following:

- Approve the PR or react with 👍 (:+1:) if it looks good to you
- Review with Comments or react with 👀 (:eyes:) if you're "ok" with it (it may not be your favorite)
- If you think it needs discussion or revision
    - Create a review, add your comments and require changes

### CODE REVIEW

**Review by aalmiray — APPROVED** [review]

File: templates/GOVERNANCE.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 ---
-status: template-draft
-instructions: Replace [Project Name], update Contact information, delete YAML frontmatter when you are good to go
+status: template
+instructions:
+  - Replace [Project Name]
+  - update Contact Information and other content to match the practices of your project
```
Comment (kenfinnigan) [inline templates/GOVERNANCE.md:5]:
```suggestion
  - Update Contact Information and other content to match the practices of your project
```

File: templates/GOVERNANCE.md [hunk 2]
```diff
@@ -1,6 +1,9 @@
...
+  - delete YAML frontmatter when you are good to go
```
Comment (kenfinnigan) [inline templates/GOVERNANCE.md:6]:
```suggestion
  - Delete YAML frontmatter when you are good to go
```

File: templates/website-footer.md [hunk 3]
```diff
@@ -1,32 +1,32 @@
 # Copyright notices for project website footers
 
-For those building or managing a website for a Commonhaus Foundation project, please incorporate one of the following standard footers.
+For those building or managing a website for a Commonhaus Foundation project, you should include mention of the Commonhaus Foundation in your website footers. Some examples are provided below (replace the links to YOUR PROJECT to point to the full URL of the main landing page for your project's website).
```
Comment (kenfinnigan) [inline templates/website-footer.md:3]: Are we requiring a Commonhaus footer, or only prefer it?

Comment (ebullient) [reply templates/website-footer.md:3]: I would like to require mention of some kind, but not necessarily this exact text or logo usage. The Trademark text should be there.

**Review by velo — APPROVED** [review]

File: templates/website-footer.md (hunk 3 above)
Comment (kenfinnigan) [reply templates/website-footer.md:3]: If we want to require usage, should the wording be:

```suggestion
For those building or managing a website for a Commonhaus Foundation project, you must include mention of the Commonhaus Foundation in your website footers. Some examples are provided below (replace the links to YOUR PROJECT to point to the full URL of the main landing page for your project's website).
```

File: templates/website-footer.md [hunk 4]
```diff
@@ -1,32 +1,32 @@
...
 
 ## Markdown
 
 ```markdown
 <img src="https://raw.githubusercontent.com/commonhaus/artwork/main/foundation/brand/svg/CF_logo_horizontal_single_default.svg" height="30" alt=""/><br />
-Copyright © [Commonhaus Foundation](https://www.commonhaus.org). All rights reserved. For details on our trademarks, please visit our [Trademark Policy](https://www.commonhaus.org/policies/trademark-policy/) and [Trademark List](https://www.commonhaus.org/trademarks/). Trademarks of third parties are owned by their respective holders and their mention here does not suggest any endorsement or association.
+Copyright © [YOUR PROEJCT](https://www.commonhaus.org). All rights reserved. For details on our trademarks, please visit our [Trademark Policy](https://www.commonhaus.org/policies/trademark-policy/) and [Trademark List](https://www.commonhaus.org/trademarks/). Trademarks of third parties are owned by their respective holders and their mention here does not suggest any endorsement or association.
```
Comment (kmccarp) [inline templates/website-footer.md:19]:
```suggestion
Copyright © [YOUR PROJECT](https://www.commonhaus.org). All rights reserved. For details on our trademarks, please visit our [Trademark Policy](https://www.commonhaus.org/policies/trademark-policy/) and [Trademark List](https://www.commonhaus.org/trademarks/). Trademarks of third parties are owned by their respective holders and their mention here does not suggest any endorsement or association.
```

**Review by marc0der — APPROVED** [review]

File: templates/website-footer.md [hunk 5]
```diff
@@ -1,32 +1,32 @@
...
 
 ## AsciiDoc
 
 ```asciidoc
 image::https://raw.githubusercontent.com/commonhaus/artwork/main/foundation/brand/svg/CF_logo_horizontal_single_default.svg[height=30]
 
-Copyright © link:https://www.commonhaus.org[Commonhaus Foundation]. All rights reserved. For details on our trademarks, please visit our link:https://www.commonhaus.org/policies/trademark-policy/[Trademark Policy] and link:https://www.commonhaus.org/trademarks/[Trademark List]. Trademarks of third parties are owned by their respective holders and their mention here does not suggest any endorsement or association.
+Copyright © link:https://www.commonhaus.org[YOUR PROJECT]. All rights reserved. For details on our trademarks, please visit our link:https://www.commonhaus.org/policies/trademark-policy/[Trademark Policy] and link:https://www.commonhaus.org/trademarks/[Trademark List]. Trademarks of third parties are owned by their respective holders and their mention here does not suggest any endorsement or association.
```
Comment (henri-tremblay) [inline templates/website-footer.md:31]: I could be wrong but for me "Copyright" = ©. So we should not have the symbol aside the word. It's one or the other (see the github footer which is using ©)

Comment (evanchooly) [reply templates/website-footer.md:31]: examples abound for using both.  i'm ok with it.

**Review by evanchooly — APPROVED** [review]

**Review by cealsair — APPROVED** [review]

**Comment by maxandersen** [comment]
a thought that occurred while reading these was if anyone actually wrote something to detect/check if the suggestions are being used.

not to check - but to simply just help those interested in bootstrapping a project?

like a `jbang ch-bootstrap@commonhaus --org jbangdev` which then would just iterate the repos and check for the presence of various files suggested?

**Pull request merged**